### PR TITLE
DOC: Update for return value of np.ptp()

### PR DESCRIPTION
**Pull request by Digya053**
Update the documentation for return type and its description of [np.ptp()](https://numpy.org/doc/stable/reference/generated/numpy.ptp.html) based on [issue#18719](https://github.com/numpy/numpy/issues/18719).

### CODE REVIEW

File: numpy/core/defchararray.py [hunk 1]
```diff
@@ -835,6 +835,17 @@ def isdigit(a):
     See Also
     --------
     str.isdigit
+
```
Comment (rossbar) [inline numpy/core/defchararray.py:838]: It looks like this change probably got committed on the wrong branch. I'm happy to help "fix" this if you'd like, just LMK!

File: numpy/core/fromnumeric.py [hunk 2]
```diff
@@ -2647,9 +2647,9 @@ def ptp(a, axis=None, out=None, keepdims=np._NoValue):
 
     Returns
     -------
-    ptp : ndarray
-        A new array holding the result, unless `out` was
-        specified, in which case a reference to `out` is returned.
+     ptp : ndarray or scalar
```
Comment (rossbar) [inline numpy/core/fromnumeric.py:2650]: Just a nit - let's get rid of the extra space that was added.
```suggestion
    ptp : ndarray or scalar
```

**Review by rossbar — COMMENTED** [review]
This generally seems like an improvement, though it looks like multiple unrelated changes snuck into the same PR.

File: numpy/core/defchararray.py (hunk 1 above)
Comment (Digya053) [reply numpy/core/defchararray.py:838]: I have added examples of isdigit on another branch as well. Yeah, please fix it.

**Comment by seberg** [comment]
Seems like all of Ross' comments were addressed and a nice improvement.  Thanks @Digya053.

**Comment by InessaPawson** [comment]
Hi-five on merging your first pull request to NumPy, @Digya053! We hope you stick around! Your choices aren’t limited to programming – you can review pull requests, help us stay on top of new and old  issues, develop educational material, work on our website, add or improve graphic design, create marketing materials, translate website content, write grant proposals, and help with other fundraising initiatives. For more info, check out: https://numpy.org/contribute
Also, consider joining our mailing list. This is a great way to connect with other cool people in our community and be part of important conversations that affect the development of NumPy: https://mail.python.org/mailman/listinfo/numpy-discussion

**Pull request merged**